package.json

```diff
@@ -28,8 +27,7 @@
   },
   "homepage": "https://github.com/misterpeddy/hands-down#readme",
   "dependencies": {
-    "express": "^4.17.1",
-    "nclr": "^2.2.4"
+    "express": "^4.17.1"
```

Comment:
I think we can actually remove `express` too?

Reply:
Yup, great catch!